src/hooks/use-form.js

```diff
@@ -116,6 +116,28 @@ const valuesReducer = (state, action) => {
   }
 };
 
+/**
+ * Field meta keys.
+ */
+
+const fieldMetaKeys = ['active', 'dirty', 'touched'];
+
+/**
+ * Get nested keys.
+ */
+
+function getNestedKeys(state: Object, allKeys: Array<string>): Array<string> {
+  return Object.keys(state).reduce((acc, key) => {
+    if (fieldMetaKeys.includes(key)) {
```

Comment:
I'm not sure about this... What if you have a field in the form named `active`?

What do you think about keeping the meta object as a flat object with keys with dots? Like this:

```js
meta: {
  'foo.bar': {
    active: false,
    // etc
  }
}
```

Let's discuss this on Monday, ok?